README.md

```diff
@@ -5,9 +5,9 @@ redux
 [![npm version](https://img.shields.io/npm/v/redux.svg?style=flat-square)](https://www.npmjs.com/package/redux)
 [![redux channel on slack](https://img.shields.io/badge/slack-redux@reactiflux-61DAFB.svg?style=flat-square)](http://www.reactiflux.com)
 
-Atomic Flux with hot reloading.  
+Atomic Flux with hot reloading.
 
-**The API is likely to change a few times before we reach 1.0.**  
```

Comment:
Whitespace is significant in Markdown. These two spaces tell it to split the lines. :-)